test/api.jl

```diff
@@ -42,4 +42,14 @@ include("utils.jl")
     end
 end
 
+@testset "Pkg.status" begin
```

Comment:
Maybe we should add the packages we call status on too (`Random` and `Dates`, and maybe swap one of them to `Example` to use a "real" package)? Also, wrap in `temp_pkg_dir() do project_path ... end` as the example above, otherwise this poisons the default env.